### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by KUBAHO3**
**In this Project I have**

- Use React documentation.
- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions and reducers in React.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,229 +61,189 @@ After you're finished please remove all the comments and instructions!
...
+If you like this project please follow me on github & twitter and also connect on Linkedin.
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- ACKNOWLEDGEMENTS -->
 
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-> Give credit to everyone who inspired your codebase.
+## :pray: Acknowledgments <a name="acknowledgements"></a>
+- I would like to thank Microverse for this exercise. <br>
```
Comment (mckent05) [inline README.md:227]: - According to the project requirements, you are required to give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Please fix this 👍 👍

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import { useNavigate } from 'react-router-dom';
+import { FaAngleLeft, FaMicrophone } from 'react-icons/fa';
+import { AiOutlineSetting } from 'react-icons/ai';
+import '../styles/Navbar.css';
+
+function Navbar() {
+  const navigate = useNavigate();
+  return (
+    <nav className="navbar-container">
+      <button className="back-button" type="button" key={Date.now()} aria-label="Back" onClick={() => navigate(-1)}><FaAngleLeft /></button>
```
Comment (mckent05) [inline src/components/Navbar.js]: **OPTIONAL** - Good job done so far, but this button on your homepage redirects outside your application, I suggest you do not display it on your homepage. Alternatively, if you must display it, You can have a different button on the detail page that redirects back to the homepage. Please fix this 🙏

File: src/pages/Home.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+import { useDispatch, useSelector } from 'react-redux';
+import { FaRegArrowAltCircleRight } from 'react-icons/fa';
+import { fetchApiData } from '../redux/metrics/metricsSlice';
+import { fetchDetailData } from '../redux/details/detailsSlice';
+import '../styles/Home.css';
+
+function Home() {
+  const dispatch = useDispatch();
+  const metrics = useSelector((store) => store.metrics.apiData);
+  const loading = useSelector((store) => store.metrics.isLoading);
```
Comment (mckent05) [inline src/pages/Home.js]: **OPTIONAL** - To help improve the readability of your code, I suggest you get the `metrics` state from your store once and destructure all your required properties. Please fix this 🙏

File: src/pages/Home.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+                  </Link>
+                </span>
+              </div>
+              <div className="image-container"><img className="metric-image" src={item.imageUrl} alt={item.image_id} /></div>
+              <div className="metric-about">
+                <h3 className="metric-title">{item.nameEU}</h3>
+                <h3 className="metric-numbers">{item.id}</h3>
+              </div>
+            </div>
+          ))
```
Comment (mckent05) [inline src/pages/Home.js:73]: - You have done well so far 👏, but when I use the search filter, the UI seems to be a little distorted, please fix this 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
- Please ensure that the background covers the entire page 😄 
ETJHAJETH

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @KUBAHO3, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job working on your capstone, it is almost ready 🚀 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Home.js (hunk 4 above)
Comment (KUBAHO3) [reply src/pages/Home.js:73]: Yes sure Thank you for the advice!

File: README.md (hunk 1 above)
Comment (KUBAHO3) [reply README.md:227]: Yes Thank you! will change that

**Review by victorokoroji — APPROVED** [review]
Hi @KUBAHO3, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested change implemented ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Home.js (hunk 3 above)
Comment (KUBAHO3) [reply src/pages/Home.js]: fixed

**Pull request merged**